glide.yaml

```diff
@@ -10,38 +10,14 @@ import:
   - service/ec2
 - package: github.com/lib/pq
 - package: github.com/motomux/pretty
-- package: k8s.io/apiextensions-apiserver
-  subpackages:
-  - pkg/client/clientset/clientset
 - package: k8s.io/apimachinery
-  version: release-1.7
-  subpackages:
-  - pkg/api/errors
-  - pkg/api/resource
-  - pkg/apis/meta/v1
-  - pkg/labels
-  - pkg/runtime
-  - pkg/runtime/schema
-  - pkg/runtime/serializer
-  - pkg/types
-  - pkg/util/intstr
-  - pkg/util/remotecommand
-  - pkg/watch
+  version: kubernetes-1.9.9
+- package: k8s.io/apiextensions-apiserver
+  version: kubernetes-1.9.9
 - package: k8s.io/client-go
-  version: ^4.0.0
-  subpackages:
-  - kubernetes
-  - kubernetes/scheme
-  - kubernetes/typed/apps/v1beta1
-  - kubernetes/typed/core/v1
-  - kubernetes/typed/extensions/v1beta1
-  - pkg/api
-  - pkg/api/v1
-  - pkg/apis/apps/v1beta1
-  - pkg/apis/extensions/v1beta1
-  - rest
-  - tools/cache
-  - tools/clientcmd
-  - tools/remotecommand
+  version: ^6.0.0
```

Comment:
what's so specific about version  6 ? afaik current docs [recommend v8](https://github.com/kubernetes/client-go#client-go)

Reply:
We want to do one step at a time, upgrade to v6, achieve our goal of implementing priority, then go further with 8.